HIP/hip-376.md

```diff
@@ -0,0 +1,155 @@
+---
+hip: 376
+title: Support for Approve/Allowance and ERC20/ERC721 transferFrom in HTS precompile
+author: Stoyan Panayotov <stoyan.panayotov@limechain.tech>
+type: Standards Track
+category: Service
+needs-council-approval: Yes
+status: Review
+created: 2022-03-02
+discussions-to: 
+updated: 
+requires: 206, 218
+---
+
+## Abstract
+
+Extend the functionality from HIP-218 for ERC20 and ERC721 funtions working on
+HTS tokens with support for approve/allowance and transferFrom functions.
+
+## Motivation
+
+The goal of this HIP is to enable users of the HSCS to interact with Hedera
```

Comment:
re-do motivation.  The goal is to expose HIP-??? functionality in industry standard ERC-20 and ERC-721 calls.